dipy/io/gradients.py

```diff
@@ -46,11 +45,10 @@ def read_bvals_bvecs(fbvals, fbvecs):
             with open(this_fname, 'r') as f:
                 content = f.read()
             # We replace coma and tab delimiter by space
-            with InTemporaryDirectory():
-                tmp_fname = "tmp_bvals_bvecs.txt"
-                with open(tmp_fname, 'w') as f:
-                    f.write(re.sub(r'(\t|,)', ' ', content))
-                vals.append(np.squeeze(np.loadtxt(tmp_fname)))
+            with tempfile.NamedTemporaryFile(suffix=".txt", mode="w") as f:
```

Comment:
Classic windows nit: https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile

> Whether the name can be used to open the file a second time, while the named temporary file is still open, varies across platforms (it can be so used on Unix; it cannot on Windows NT or later).

I'll swap out this implementation